impress-org/givewp
# Feature: add create new campaign flow

Resolves [GIVE-1217]

## Description



This PR split the form to create a new campaign in two steps and add all necessary fields and styles to make it work as expected.  The main goal here is to implement the functionality that allows us to run some internal tests, but we'll need to tweak some things later related to design adjustments and other improvements.

**Important:** Please note that I'm planning to open a subsequent PR when PR #7547 gets merged to the Epic, so I'll be able to refactor this form to use the new `@wordpress/core-data approach` used there and also will replace the Goal input to use a CurrencyInput (which is being introduced in the mentioned PR) when the "Amount raised" option is selected on the Goal Type options.

## Affects



The "Create New Campaign" form.

## Visuals



![image](https://github.com/user-attachments/assets/a51aae58-ed36-4c5a-ba96-f2179c1ce7c3)

![image](https://github.com/user-attachments/assets/19e1fdae-5041-4aff-8a68-be229c744e44)


## Testing Instructions



1. Go to `Give > Campaigns` page;
2. Click on the `Create campaign` button;
3. Follow the steps to create a new campaign;
4. Make sure everything worked as expected.

## Pre-review Checklist






[GIVE-1217]: https://stellarwp.atlassian.net/browse/GIVE-1217?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
